interop/client/client.go

```diff
@@ -49,6 +49,7 @@ import (
 var (
 	useTLS                = flag.Bool("use_tls", false, "Connection uses TLS if true, else plain TCP")
 	testCA                = flag.Bool("use_test_ca", false, "Whether to replace platform root CAs with test CA as the CA root")
+	testCACustomFile      = flag.String("use_custom_test_ca", "", "If provided, replaces platform root CAs with custom CA file")
```

Comment:
This needs to also be updated in the doc, since these flags were standardized across all languages:

https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md